api/pkg/transformer/pipeline/upi_postprocess_output_op_test.go

```diff
@@ -255,7 +255,7 @@ func TestUPIPostprocessOutputOp_Execute(t *testing.T) {
 				PredictionResultTableName: "driver_table_not_exist",
 			},
 			env:    env,
-			expErr: fmt.Errorf("table driver_table_not_exist is not declared"),
+			expErr: fmt.Errorf("invalid input: table driver_table_not_exist is not declared"),
```

Comment:
should we quote all the table and column names? i.e. "table `driver_table_not_exist` is not declared"